tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "@tsconfig/node14/tsconfig.json",
```

Comment:
Suggestion to have this use Node 16 since we want all apps to move to that now anyway.

Reply:
Need to upgrade this to Node 16.